test/cadet/accounts/accounts_test.exs

```diff
@@ -134,73 +134,46 @@ defmodule Cadet.AccountsTest do
         user: user
       })
 
-      assert {:ok, user} == Accounts.sign_in("e012345", @token)
+      assert {:ok, user} == Accounts.sign_in("e012345", "TOM", @token)
     end
 
     test "invalid token" do
       use_cassette "accounts/sign_in#2" do
-        assert {:error, :bad_request} == Accounts.sign_in("A0123456", "t0k3n")
+        assert {:error, :bad_request} == Accounts.sign_in("A0123456", "TOM", "t0k3n")
       end
     end
 
     test "invalid nusnet id" do
       use_cassette "accounts/sign_in#3" do
-        assert {:error, :internal_server_error} == Accounts.sign_in("", @token)
+        assert {:error, :internal_server_error} == Accounts.sign_in("", "TOM", @token)
```

Comment:
By right, invalid NUSNET id should return `:bad_request` right?

Reply:
Hmmm I think it should be a 500 internal server error leh. Since nusnet id comes directly from luminus it should never be invalid or wrong. If it is either our server or their server has an inconsistent state. haha but im open to throw a bad request too.